File: app/src/main/java/com/ilustris/motiv/adapters/HomePager.java
package com.ilustris.motiv.adapters;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import com.ilustris.motiv.fragments.HomeFragment;
import com.ilustris.motiv.fragments.PicsFragment;

public class HomePager extends FragmentPagerAdapter {
    public HomePager(@NonNull FragmentManager fm) {
        super(fm);
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        if (position == 0){
            return  new HomeFragment();
        }else{
            return new PicsFragment();
        }
    }

    @Override
    public int getCount() {
        return 2;
    }
}

File: app/src/main/java/com/ilustris/motiv/adapters/HomePager.kt
package com.ilustris.motiv.adapters

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

import com.ilustris.motiv.fragments.HomeFragment
import com.ilustris.motiv.fragments.PicsFragment
import com.ilustris.motiv.fragments.SuggestionsFragment

class HomePager(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return when (position) {
            0 -> HomeFragment()
            1 -> PicsFragment()
            else -> SuggestionsFragment()
        }
    }

    override fun getCount(): Int {
        return 3
    }
}
